Makefile

```diff
@@ -178,7 +180,7 @@ $(DOCKER_DEVKIT_PHONY_FILE): Dockerfile.devkit
 	&& touch $(DOCKER_DEVKIT_PHONY_FILE)
 
 $(DOCKER_PHONY_FILE): $(DOCKER_DEVKIT_PHONY_FILE)
-$(DOCKER_PHONY_FILE): bin/konvoy-image
+$(DOCKER_PHONY_FILE): konvoy-image-linux
```

Comment:
creating linux binary specifically so that mac users dont have to set GOOS=linux explicitly when building the kib devkit image.